java-lib/src/main/java/org/clink/feature/onehotencoder/ClinkOneHotEncoder.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2021 The Clink Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.clink.feature.onehotencoder;
+
+import org.apache.flink.ml.api.Estimator;
+import org.apache.flink.ml.feature.onehotencoder.OneHotEncoder;
+import org.apache.flink.ml.feature.onehotencoder.OneHotEncoderModel;
+import org.apache.flink.ml.feature.onehotencoder.OneHotEncoderParams;
+import org.apache.flink.ml.param.Param;
+import org.apache.flink.ml.util.ReadWriteUtils;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.api.Table;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class ClinkOneHotEncoder
+        implements Estimator<ClinkOneHotEncoder, ClinkOneHotEncoderModel>,
+                OneHotEncoderParams<ClinkOneHotEncoder> {
+    private final OneHotEncoder estimator;
```

Comment:
I think we need to forward parameters like dropLast to `estimator`, since it is used in `estimator.fit()`.

Furthermore, I think all parameters received by the outer `ClinkOneHotEncoder` should be sent to `estimator`, as they both implement `OneHotEncoderParams`.

My consideration behind this is to use delegation instead of inheritance, given that `ClinkOneHotEncoder` and `OneHotEncoder` shares much functions in common but they do not have the same method signatures.